spv/envelope.go

```diff
@@ -9,13 +12,12 @@ import (
 
 // Envelope is a struct which contains all information needed for a transaction to be verified.
 //
-// spec at https://tsc.bitcoinassociation.net/standards/spv-envelope/
 type Envelope struct {
-	TxID          string               `json:"txid,omitempty"`
-	RawTx         string               `json:"rawTx,omitempty"`
-	Proof         *bc.MerkleProof      `json:"proof,omitempty"`
-	MapiResponses []bc.MapiCallback    `json:"mapiResponses,omitempty"`
-	Parents       map[string]*Envelope `json:"parents,omitempty"`
+	TxID          string
```

Comment:
why remove the json tags? they'd still be useful and don't affect anything do they?

Reply:
I was thinking it would clarify that the purpose of the struct is internal, but to be fair now that I know you're keen on the json for debugging - I see the value you're referring to. Will put them back!